client.py

```diff
@@ -1275,8 +1275,8 @@
 			market_data.persist()
 
 			if usermodel != None:
-					
-				usermodel.slimecoin = max(0, (usermodel.slimecoin - (usermodel.swear_jar * swear_multiplier * 10000))) # fine the user for swearing, 1 billion slimecoin for every swear in their swear jar
+				# fine the user for swearing, based on how much they've sworn right now, as well as in the past
+				usermodel.change_slimecoin(n= -1 * (usermodel.swear_jar * swear_multiplier * 10000), coinsource=ewcfg.coinsource_swearjar)
```

Comment:
change_slimecoin doesn't actually make it so value bottoms out at 0. you can have negative slimecoin this way. though that may be funny in its own way

Reply:
I don't even wanna fucking think about what could happen as a result of that, so I'll go ahead and fix this.